color/src/color.rs

```diff
@@ -404,8 +397,7 @@ impl<CS: ColorSpace> AlphaColor<CS> {
 
     /// Map the hue of the color.
     ///
-    /// In a color space that naturally has a hue component, map that value.
-    /// Otherwise, do the mapping in [Oklch]. The hue is in degrees.
+    /// See [`OpaqueColor::map_hue`] for more details.
     #[must_use]
     pub fn map_hue(self, f: impl Fn(f32) -> f32) -> Self {
```

Comment:
This is orthogonal, but is there some name we could give this function to hint to the user that the value is in degrees? (i.e. in inlay hints).